webapp/src/Controller/RootController.php

```diff
@@ -45,6 +45,10 @@ public function redirectAction(AuthorizationCheckerInterface $authorizationCheck
                 return $this->redirectToRoute('jury_balloons');
             }
         }
+        $this->addFlash(
+            'danger',
+            sprintf("User has no DOMjudge roles, user deauthenticated.)
```

Comment:
Why a sprintf? Also I think you are missing a “

Reply:
I should indeed remove that.